R/test-that.r

```diff
@@ -73,6 +82,28 @@ test_code <- function(description, code, env) {
   invisible(ok)
 }
 
+report_results <- function(results) {
```

Comment:
I think this would be better called `report_expectation()`, but I'm not sure `report()` is quite the right verb (but I'm not sure what would be better)